web/templates/settings/bot_list_admin.hbs

```diff
@@ -1,4 +1,4 @@
-<div id="admin-bot-list" class="settings-section" data-name="bot-list-admin">
+<div id="admin-bot-list" class="settings-section" data-name="bots">
```

Comment:
I am not sure but we might add code to redirect `bots-list-admin` to `bots` in hashchange.